addons/map_gestures/functions/fnc_addGroupColorMapping.sqf

```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: Dslyecxi, MikeMatrix, PabstMirror
+ * Adds a group id color mapping.
+ *
+ * Arguments:
+ * 0: Group ID or group <STRING><GROUP>
+ * 1: Leader color array (4 numbers including alpha) <ARRAY>
+ * 2: Unit (non-leader) color array (4 numbers including alpha) <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["Alpha 1-1", [1,0,0,1], [0,1,0,1]] call ace_map_gestures_fnc_addGroupColorMapping
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+if (!params [["_group", "", [grpNull, ""]], ["_leadColor", [1,1,1,1], [[]], 4], ["_unitColor", [1,1,1,1], [[]], 4]]) exitWith {
+    ERROR_1("Bad Params: %1",_this);
+    -1
```

Comment:
returning `-1` here but on the other `exitWith` statements we do not return anything.